### PR TITLE
Blog app - Views 📸 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# What I have done in this part of project 🥇 

- Implement the design from the sneak peek wireframes
- Styling
- Use methods that I have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
# 🧠 Skills I gained from this project 🌳 
- [ ]  Use preprocessed HTML file with embedded Ruby code.
- [ ]  Use layouts and templates for shared content.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @recent_posts = User.all.map(&:recent_posts).flatten
```
Comment (descholar-ceo) [inline app/controllers/users_controller.rb]: - [ ] We are supposed to return all users in this action
```suggestion
    @users = User.all
```

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1 +1,12 @@
...
+<div>
+  <% @recent_posts.each do |post| %>
+  <ul>
+    <li><img src=<%= post.author.photo %>></li>
+    <div class="border"> 
+      <li><%= post.author.name %></li>
+      <li>Number of posts <%= post.author.posts_counter %></li>
+    </div>
+  </ul>
+<% end %>
```
Comment (descholar-ceo) [inline app/views/users/index.html.erb]: - [ ] We are supposed to render all users here
```suggestion
 <% @users.each do |user| %>
     # HTML Code Here displaying user.photo, user.name and user.posts_counter
    <% end %>
```

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @ismailmunyentwari9 🤝 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
👍🏽 PR is professional with a title and a description ✔️ 
👍🏽 The linters are installed and the PR passes the check ✔️ 
👍🏽 The commit messages are professional ✔️ 
👍🏽 Readme file follows microverse's template ✔️ 

## Required Changes ♻️
_Check the comments under the review._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @ismailmunyentwari9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Clean code
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**